### ISSUE TITLE
no such identity: ~/.ssh/id_ed25519: No such file or directory

### ISSUE THREAD

**Issue by mikaeleiman** [KryptCo/kr#84]
During the install kr added a bunch of IdentityFile lines to ~/.ssh/config. One of the lines references id_ed25519, but that file doesn't exist. For some reason ssh complains about this (but not the other non-existing files) all the time.

MacOS 10.12.4.

**Comment by kcking** [comment]
The easiest way to fix this is to generate an unused ed25519 key by running `ssh-keygen -t ed25519`

We add these lines to make sure that SSH presents the kryptonite key, but also falls back to any local keys the user might have if the kryptonite key does not have access.

**Comment by mikaeleiman** [comment]
Couldn't the adding be made optional so you only add files that actually exist?

I've commented out the offending line in config, could that cause any problems?

**Comment by kcking** [comment]
We could do that but we opted for only modifying the config file once. If a user happened to generate a new type of key later, it would never be offered by SSH unless we detected this and added the line. We saw this error message as the best tradeoff.

Commenting it out won't cause any problems, except if you install the find and replace command to remove our config block won't detect it and you'd just have to remove it manually.
